packages/pie/src/Pie.js

```diff
@@ -1,24 +1,19 @@
-/*
- * This file is part of the nivo project.
- *
- * Copyright 2016-present, Raphaël Benitte.
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-import React, { Component } from 'react'
-import compose from 'recompose/compose'
-import defaultProps from 'recompose/defaultProps'
-import pure from 'recompose/pure'
-import withPropsOnChange from 'recompose/withPropsOnChange'
-import setDisplayName from 'recompose/setDisplayName'
+// /*
```

Comment:
Ah, this was because of my method for conversion. I commented out the original component (control + a, then control + /) and remade it piece by piece. Fixed it now.